modules/gitops-argocd-cli-utility-commands.adoc

```diff
@@ -0,0 +1,104 @@
+// Module is included in the following assemblies:
+//
+// * gitops_cli_argocd/argocd-gitops-cli-reference.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="gitops-argocd-cli-utility-commands_{context}"]
+= Utility commands
+
+== argocd
+Parent command for {gitops-shortname} `argocd` CLI.
+
+.Example: Display all options
+[source,terminal]
+----
+$ argocd
+----
+
+== version
+Print version information of the CLI.
+
+.Command syntax
+`argocd version [flags]`
+
+.Example: Print the full version of client and server to stdout
+[source,terminal]
+----
+$ argocd version
+----
+
+.Example: Print only full version of the client, no connection to server will be made
+[source,terminal]
+----
+$ argocd version --client
+----
+
+.Example: Print only full version of the server
+[source,terminal]
+----
+$ argocd version --server openshift-gitops.openshift-gitops.crc.local
+----
+
+.Example: Print the full version of client and server in JSON format
+[source,terminal]
+----
+$ argocd version -o json
+----
+
+.Example: Print only client and server core version strings in YAML format
+[source,terminal]
+----
+$ argocd version --short -o yaml
+----
+
+== help
+Print the help message about any command in the application.
+
+.Command syntax
+`argocd help [command] [flags]`
+
+.Example: Get the help text for all the available commands
+[source,terminal]
+----
+$ argocd help
+----
+
+.Example: Get the help text for `admin` subcommand
+[source,terminal]
+----
+$ argocd help admin
+----
+
+== completion 
+Write `bash` or `zsh` shell completion code to standard output.
```

Comment:
Why do we only document the usage of Bash shell completion if it exists for both Bash and Zsh?

Reply:
@anandf Can we have the usage for Zsh too?